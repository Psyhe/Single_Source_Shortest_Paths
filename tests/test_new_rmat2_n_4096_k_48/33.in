4096 2821 2905
2821 2565 106
2821 2881 150
2821 2661 67
2821 3841 58
2821 821 175
2821 1285 69
2821 2817 44
2821 661 226
2821 901 254
2821 2579 93
2822 550 16
2822 530 104
2822 2894 19
2822 1807 156
2822 773 251
2822 2846 217
2822 2850 106
2822 2830 117
2822 2063 76
2822 2438 65
2822 6 187
2822 2948 91
2822 2326 150
2822 2438 188
2822 2246 238
2822 2844 118
2822 854 19
2822 2830 70
2822 3862 212
2822 3590 63
2822 3846 193
2823 1919 44
2823 2582 116
2823 2951 250
2823 781 159
2823 2951 104
2823 3845 254
2823 1923 151
2823 711 153
2823 2819 109
2824 2816 131
2824 792 77
2824 2568 196
2824 776 86
2824 2944 12
2824 2888 161
2824 2352 128
2824 18 10
2824 328 139
2824 810 157
2824 3849 187
2824 904 40
2824 2896 45
2824 3080 92
2824 2825 199
2824 2968 50
2824 2880 159
2824 2856 41
2824 2816 162
2824 520 19
2824 869 206
2824 904 184
2824 256 222
2824 2568 248
2824 2956 253
2824 2848 86
2825 1829 235
2825 2761 61
2825 2329 87
2825 2568 103
2825 2573 228
2825 1545 254
2825 1800 51
2825 3593 147
2825 2826 41
2825 2888 248
2825 2057 80
2825 3339 108
2825 3881 204
2825 2905 119
2825 2824 199
2825 811 138
2825 843 26
2825 1803 42
2826 2856 173
2826 2330 190
2826 2131 41
2826 2307 97
2826 2987 145
2826 2859 152
2826 2346 211
2826 2315 203
2826 1800 137
2826 2825 41
2826 578 231
2826 2334 189
2826 2602 195
2826 780 8
2826 2570 102
2826 3442 141
2826 2304 45
2826 2445 53
2826 2959 144
2826 2313 77
2826 826 229
2826 2314 209
2826 2318 112
2827 2831 70
2827 2945 170
2827 1407 177
2827 3591 234
2827 2841 161
2827 3842 179
2827 2057 60
2827 2891 205
2827 795 110
2827 3865 79
2828 782 225
2828 2845 202
2828 3462 52
2828 2862 191
2828 260 225
2828 44 127
2828 776 163
2828 2856 218
2828 2312 99
2828 2574 111
2828 1820 174
2828 517 92
2828 2892 205
2828 2445 190
2828 2061 74
2828 268 141
2828 2829 5
2828 780 109
2828 2966 110
2828 1944 205
2829 2313 241
2829 301 34
2829 2951 120
2829 2621 156
2829 2570 206
2829 3109 254
2829 2377 21
2829 2828 5
2829 516 93
2829 2061 196
2829 2861 68
2830 3982 112
2830 2990 83
2830 2874 18
2830 2822 117
2830 2831 0
2830 2334 127
2830 2822 70
2830 2374 116
2830 2702 62
2830 781 176
2830 911 101
2830 2886 223
2831 2827 70
2831 2830 0
2831 815 49
2831 2893 115
2831 2927 86
2831 398 92
2831 381 161
2831 2763 174
2832 2580 120
2832 2961 227
2832 768 198
2832 2617 157
2832 529 97
2832 2592 139
2832 2320 69
2832 2576 41
2832 256 101
2832 2616 241
2832 2304 248
2832 520 101
2832 3220 185
2832 2818 75
2832 2880 211
2832 2640 88
2832 2596 50
2832 960 248
2832 913 20
2832 2385 212
2832 3856 179
2832 2384 158
2832 536 164
2832 2320 96
2832 3862 239
2832 1808 121
2833 784 120
2833 18 56
2833 2840 69
2833 2368 212
2833 769 117
2833 560 189
2833 817 227
2833 2817 126
2833 2448 46
2833 2323 141
2833 3217 97
2833 2973 14
2833 445 9
2833 3891 168
2833 1840 67
2833 1303 40
2834 2950 154
2834 2948 203
2834 466 112
2834 2818 204
2834 2678 66
2834 2626 112
2834 3870 90
2834 2850 164
2834 2309 27
2834 1092 51
2834 850 140
2834 848 90
2834 2642 226
2834 2098 52
2834 2320 9
2834 3330 20
2834 2643 54
2834 2842 25
2835 3699 226
2835 2582 190
2835 2611 177
2835 1429 252
2835 2562 70
2835 2816 46
2835 2819 60
2835 634 121
2835 2486 48
2836 2900 177
2836 516 183
2836 3988 233
2836 3356 72
2836 2964 223
2836 529 104
2836 2868 56
2836 2644 125
2836 2308 213
2836 788 175
2836 2572 3
2836 524 58
2836 3892 232
2836 2944 121
2836 789 169
2836 2964 184
2836 2924 119
2836 772 186
2836 2396 29
2836 806 175
2837 2564 113
2837 769 161
2837 2933 194
2837 541 150
2837 2908 44
2837 342 160
2837 3093 73
2837 2965 145
2837 2612 171
2837 2989 247
2837 2630 95
2837 2308 182
2837 2709 101
2838 2326 176
2838 2902 23
2838 2212 251
2838 791 148
2838 2068 200
2838 2068 42
2838 786 1
2838 918 73
2838 2582 110
2838 2868 49
2838 3847 42
2838 518 0
2838 2438 189
2838 3868 160
2839 3929 173
2839 2309 230
2839 2711 159
2839 2978 16
2839 2819 177
2839 2854 180
2840 600 144
2840 2846 217
2840 2904 212
2840 920 89
2840 2833 69
2840 2129 184
2840 4040 255
2840 2316 254
2840 280 254
2840 858 123
2840 1848 143
2840 2576 211
2840 2392 141
2840 2906 232
2840 3864 15
2840 3768 238
2840 2332 239
2841 2600 132
2841 2699 221
2841 2577 234
2841 669 2
2841 2872 212
2841 905 228
2841 2201 254
2841 2827 161
2841 2360 61
2841 3864 172
2841 2577 245
2841 2329 155
2842 2712 85
2842 3903 11
2842 2706 49
2842 2971 95
2842 178 162
2842 858 137
2842 2952 57
2842 3864 207
2842 2582 43
2842 810 211
2842 826 33
2842 2816 220
2842 2834 25
2842 1922 169
2842 810 30
2842 830 182
2843 2847 111
2843 2683 18
2843 793 210
2843 2361 103
2843 2846 16
2844 2084 12
2844 2329 10
2844 3073 193
2844 3028 76
2844 2822 118
2844 3866 81
2844 850 105
2844 2716 26
2844 2684 43
2844 2584 121
2844 312 94
2844 2392 53
2844 892 87
2844 3644 170
2844 2652 199
2845 2828 202
2845 541 249
2845 596 77
2845 2868 200
2845 2685 121
2845 349 13
2845 2973 199
2845 2333 116
2845 2617 107
2846 2710 22
2846 2840 217
2846 3863 92
2846 2908 108
2846 27 87
2846 2843 16
2846 2334 30
2846 2870 219
2846 2822 217
2846 2750 14
2847 783 72
2847 2334 41
2847 2867 175
2847 2843 111
2848 2976 169
2848 514 81
2848 288 124
2848 2948 213
2848 2464 100
2848 866 230
2848 2824 86
2848 2400 112
2848 2338 100
2848 2849 223
2848 768 41
2848 2976 224
2848 2144 76
2848 3616 45
2848 3873 187
2848 2944 241
2848 137 236
2848 3605 84
2848 2304 101
2848 2726 18
2848 2720 75
2848 3464 14
2848 2984 129
2849 801 210
2849 2921 225
2849 2848 223
2849 2605 109
2849 3380 105
2849 2344 229
2849 2609 9
2849 801 179
2849 359 62
2849 2817 123
2849 2656 71
2849 545 94
2849 2210 249
2849 273 74
2849 33 185
2850 2722 51
2850 2656 236
2850 2834 164
2850 2458 255
2850 818 219
2850 294 217
2850 290 125
2850 2344 245
2850 2874 132
2850 1314 209
2850 2720 230
2850 800 236
2850 2434 102
2850 3619 49
2850 2822 106
2850 3890 114
2850 38 36
2850 2912 231
2850 3042 149
2850 2859 188
2850 2362 0
2851 2435 27
2851 2659 221
2851 802 3
2851 2976 197
2851 2082 164
2851 359 7
2851 771 159
2851 2857 47
2851 2339 165
2851 2976 40
2851 3750 215
2851 1803 112
2851 2867 85
2851 47 41
2851 3905 26
2852 2917 80
2852 2212 94
2852 3364 255
2852 2628 132
2852 2183 120
2852 822 154
2852 2916 147
2852 3636 253
2852 2854 91
2852 3075 53
2852 805 119
2853 4009 85
2853 772 8
2853 260 216
2853 3669 25
2853 2900 215
2853 1285 91
2853 2977 90
2853 2309 187
2853 807 164
2854 3936 91
2854 2982 37
2854 2852 91
2854 2839 180
2854 807 191
2854 2338 6
2855 2595 18
2855 2566 132
2855 3847 29
2855 2918 136
2855 2978 118
2855 2950 148
2856 296 213
2856 2152 3
2856 553 42
2856 2352 44
2856 3112 244
2856 2344 238
2856 712 199
2856 4001 133
2856 810 146
2856 1596 128
2856 3692 200
2856 776 156
2856 2828 218
2856 266 162
2856 551 132
2856 2824 41
2856 3864 156
2856 2826 173
2856 2572 126
2856 4012 114
2856 2090 19
2856 2080 43
2857 4009 14
2857 3873 50
2857 2851 47
2857 2976 30
2857 2669 147
2857 2105 128
2857 2060 56
2857 2408 223
2857 2089 214
2857 2616 6
2857 889 214
2857 3897 126
2857 3373 156
2857 2859 49
2858 2602 161
2858 2314 101
2858 2862 35
2858 3350 173
2858 2634 99
2858 2601 179
2858 362 25
2858 1282 194
2858 2986 65
2858 2602 225
2858 2370 166
2858 554 111
2858 2918 71
2859 3979 206
2859 3219 63
2859 2603 73
2859 3883 216
2859 387 239
2859 1579 184
2859 3055 3
2859 2857 49
2859 2850 188
2859 2826 152
2859 2922 135
2860 296 232
2860 2925 206
2860 1388 201
2860 2989 200
2860 2333 10
2860 2601 53
2860 2878 212
2860 2924 34
2860 2345 79
2860 2348 3
2861 2305 124
2861 1849 218
2861 2989 45
2861 2493 77
2861 836 9
2861 2873 18
2861 2683 29
2861 2415 212
2861 2829 68
2861 3877 30
2862 2078 11
2862 2350 195
2862 948 147
2862 2828 191
2862 3374 60
2862 550 215
2862 2318 205
2862 2093 79
2862 2995 78
2862 2350 157
2862 2858 35
2863 782 218
2863 2613 5
2863 1839 123
2863 2494 91
2863 1829 213
2864 2305 98
2864 2865 3
2864 2976 255
2864 380 129
2864 2736 159
2864 3000 210
2864 2576 20
2864 2616 32
2864 2362 200
2864 308 74
2865 385 9
2865 785 224
2865 2864 3
2865 3057 177
2865 4081 216
2865 2929 33
2865 864 49
2865 825 134
2866 2914 144
2866 2327 210
2866 2354 169
2866 2690 54
2866 2818 178
2866 2870 7
2866 951 45
2866 2870 239
2867 1395 116
2867 2847 175
2867 2101 243
2867 2967 162
2867 2851 85
2867 951 82
2867 787 72
2868 3889 67
2868 2613 206
2868 3380 48
2868 2980 216
2868 950 135
2868 660 252
2868 3638 87
2868 432 38
2868 2836 56
2868 2838 49
2868 564 121
2868 2845 200
2868 2871 182
2868 774 38
2869 3873 168
2869 916 20
2869 2928 209
2869 2577 253
2869 2997 112
2869 2877 26
2869 1889 43
2869 800 198
2869 3061 119
2870 1844 221
2870 306 92
2870 3900 211
2870 3128 180
2870 772 191
2870 2846 219
2870 1526 123
2870 2612 247
2870 2358 111
2870 2866 7
2870 2866 239
2870 2686 3
2871 309 11
2871 935 61
2871 279 132
2871 2868 182
2871 2938 23
2871 2917 112
2872 2785 154
2872 816 176
2872 2952 184
2872 2360 232
2872 2841 212
2872 3897 181
2872 280 129
2872 3896 137
2872 2961 138
2872 63 153
2872 786 107
2872 2360 35
2873 2345 28
2873 2569 183
2873 3073 223
2873 3769 175
2873 3417 134
2873 888 43
2873 2929 115
2873 825 103
2873 369 247
2873 3632 251
2873 2861 18
2874 2830 18
2874 2930 54
2874 3899 144
2874 480 119
2874 2616 135
2874 2906 36
2874 2850 132
2874 809 103
2874 3898 151
2875 2354 207
2875 3613 75
2875 58 141
2876 2632 153
2876 1792 174
2876 2620 2
2876 2877 167
2876 2348 75
2876 428 49
2877 3005 131
2877 1305 27
2877 949 185
2877 2869 26
2877 3869 125
2877 877 153
2877 2492 6
2877 2876 167
2878 2860 212
2878 2750 170
2878 542 158
2878 2110 238
2878 3390 35
2878 1694 129
2878 2622 0
2878 826 66
2880 2824 159
2880 2944 186
2880 2896 182
2880 840 221
2880 2817 168
2880 2368 104
2880 2320 94
2880 2768 180
2880 770 42
2880 448 133
2880 2881 117
2880 833 243
2880 2564 58
2880 96 198
2880 840 76
2880 3884 43
2880 778 168
2880 836 45
2880 2832 211
2880 2372 200
2880 2653 226
2880 2368 248
2880 2896 180
2880 768 50
2880 2376 81
2880 320 11
2880 3397 199
2880 2624 103
2880 2882 226
2881 3013 218
2881 1017 36
2881 2821 150
2881 2307 226
2881 1893 49
2881 3889 24
2881 2913 77
2881 3009 29
2881 515 1
2881 2880 117
2881 2897 110
2881 3041 80
2881 513 35
2881 837 159
2881 3016 136
2881 2884 186
2881 2883 144
2881 1795 5
2881 3921 246
2881 800 174
2881 2883 214
2881 779 21
2881 2625 160
2881 2631 34
2881 3584 62
2881 2740 87
2882 1386 161
2882 2880 226
2882 3010 6
2882 2910 253
2882 2627 85
2882 2898 101
2882 3010 23
2882 2640 178
2882 2114 138
2882 2928 1
2882 3942 21
2882 2883 136
2882 514 178
2882 2626 35
2882 0 2
2882 2894 139
2882 2376 46
2882 3011 66
2883 331 204
2883 2571 173
2883 2629 45
2883 2881 144
2883 2375 234
2883 2881 214
2883 2891 221
2883 771 194
2883 451 59
2883 2371 21
2883 2896 57
2883 33 160
2883 1859 82
2883 2882 136
2884 2892 105
2884 68 79
2884 833 184
2884 260 143
2884 2628 190
2884 3020 111
2884 1856 255
2884 4032 91
2884 2881 186
2884 2646 31
2884 2886 144
2884 580 220
2884 2820 214
2885 3349 211
2885 2629 195
2885 2369 221
2885 3397 249
2885 2645 193
2885 2656 44
2885 2797 13
2885 2887 227
2885 2945 241
2885 3531 155
2885 3457 44
2886 591 89
2886 2830 223
2886 3996 99
2886 2004 62
2886 2438 225
2886 2884 144
2886 294 45
2886 2902 162
2886 1990 86
2886 2927 172
2887 83 170
2887 2885 227
2887 615 32
2887 2631 135
2887 2629 244
2887 2935 201
2887 2756 247
2887 3009 159
2888 2392 32
2888 2825 248
2888 2648 122
2888 2824 161
2888 2952 82
2888 2572 152
2888 1864 179
2888 3648 58
2888 2666 178
2888 2376 99
2888 2496 150
2888 473 162
2888 3936 207
2888 2056 167
2888 2626 102
2888 2633 167
2889 2985 185
2889 1097 178
2889 3851 144
2889 3065 159
2889 2633 137
2889 2369 75
2889 2683 66
2889 840 75
2889 2539 27
2889 3397 87
2889 281 95
2889 845 184
2889 3883 168
2890 2496 250
2890 3866 249
2890 2922 210
2890 900 230
2890 3019 87
2890 3658 104
2890 3596 72
2890 2632 83
2890 855 106
2890 1864 152
2890 3147 78
2891 3019 215
2891 1871 210
2891 2895 121
2891 2827 205
2891 2883 221
2891 863 23
2892 868 131
2892 428 95
2892 2662 3
2892 780 154
2892 2893 23
2892 2376 16
2892 2828 205
2892 2664 22
2892 2884 105
2892 588 5
2892 2572 222
2892 2649 79
2892 2316 141
2892 292 123
2893 2831 115
2893 2892 23
2893 845 177
2893 847 81
2893 2399 145
2894 2882 139
2894 3038 14
2894 2383 194
2894 2895 170
2894 3053 128
2894 1984 202
2894 2958 213
2894 2414 167
2894 2822 19
2895 2399 193
2895 2651 121
2895 2655 84
2895 2526 58
2895 2894 170
2895 2575 169
2895 843 143
2895 2891 121
2895 2783 251
2895 847 203
2895 2503 11
2895 2991 239
2896 2064 57
2896 768 166
2896 2128 99
2896 2883 57
2896 3392 231
2896 2672 32
2896 2880 182
2896 3336 96
2896 2900 107
2896 840 186
2896 2904 45
2896 3924 142
2896 2897 249
2896 850 126
2896 2900 177
2896 2712 250
2896 791 173
2896 2880 180
2896 816 244
2896 2900 105
2896 2968 239
2896 2824 45
2896 976 31
2897 3911 169
2897 3409 105
2897 2641 32
2897 2561 214
2897 2896 249
2897 2937 10
2897 2881 110
2897 594 21
2897 2499 167
2897 2672 95
2897 881 99
2898 2322 193
2898 2322 120
2898 3408 126
2898 2903 82
2898 592 72
2898 2386 212
2898 3026 235
2898 2968 55
2898 2496 28
2898 2642 14
2898 2514 122
2898 2882 101
2899 2910 152
2899 2907 54
2899 962 44
2899 1107 171
2899 851 15
2900 2622 24
2900 2853 215
2900 3028 19
2900 2908 240
2900 2896 177
2900 2132 129
2900 3932 200
2900 2896 105
2900 2641 191
2900 3924 121
2900 2908 76
2900 860 94
2900 2836 177
2900 2818 90
2900 2896 107
2900 3908 178
2901 340 234
2901 3412 100
2901 2327 14
2901 869 169
2902 2903 19
2902 2096 201
2902 2312 111
2902 3415 231
2902 84 20
2902 854 156
2902 594 198
2902 2886 162
2902 2911 24
2902 2838 23
2902 3328 8
2903 2943 177
2903 839 147
2903 2902 19
2903 2390 97
2903 2898 82
2903 2627 87
2903 1559 145
2904 2386 75
2904 2840 212
2904 3024 9
2904 2909 250
2904 873 36
2904 2896 45
2905 2433 160
2905 2649 104
2905 872 130
2905 2825 119
2905 3009 79
2905 3920 211
