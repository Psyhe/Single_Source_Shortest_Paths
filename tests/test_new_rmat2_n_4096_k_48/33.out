80
58
144
30
56
28
103
105
100
98
98
76
32
30
61
36
113
24
97
70
54
55
73
29
66
64
90
59
49
58
31
92
49
50
40
34
55
84
104
36
32
49
95
44
41
72
116
71
64
79
81
81
50
116
106
37
61
65
inf
26
26
2
30
92
85
77
122
83
89
113
196
42
65
77
99
61
61
72
71
61
55
44
63
82
134
